composer.json

```diff
@@ -22,19 +22,14 @@
         "news": "nntp://lists.typo3.org"
     },
     "require": {
-        "typo3/cms-backend": "9.5.*@dev",
-        "typo3/cms-core": "9.5.*@dev",
-        "typo3/cms-extbase": "9.5.*@dev",
-        "typo3/cms-extensionmanager": "9.5.*@dev",
-        "typo3/cms-filelist": "9.5.*@dev",
-        "typo3/cms-fluid": "9.5.*@dev",
-        "typo3/cms-frontend": "9.5.*@dev",
-        "typo3/cms-install": "9.5.*@dev",
-        "typo3/cms-recordlist": "9.5.*@dev"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "9.5.x-dev"
```

Comment:
The branch alias also does not harm. It should be changed to 10.0.x-dev once TYPO3 9.5 is branched.